tests/unit-ngv/test_ngv.py

```diff
@@ -0,0 +1,75 @@
+
+from ..conftest import NGV_DIR
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+
+def get_Rad(astro_id, manager):
+    astrocyte = manager._cell_manager.gid2cell[astro_id + manager._gid_offset]
+    if astrocyte.endfeet is None:
+        return []
+    return [sec(0.5).vascouplingB.Rad for sec in astrocyte.endfeet]
+
+
+def get_R0pas(astro_id, manager):
+    astrocyte = manager._cell_manager.gid2cell[astro_id + manager._gid_offset]
+    if astrocyte.endfeet is None:
+        return []
+    return [sec(0.5).vascouplingB.R0pas for sec in astrocyte.endfeet]
+
+
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "src_dir": str(NGV_DIR),
+        "simconfig_file": "simulation_config.json"
+    }
+], indirect=True)
+def test_vasccouplingB_radii(create_tmp_simulation_config_file):
+    from neurodamus import Neurodamus
+    from neurodamus.ngv import GlioVascularManager
```

Comment:
As per discussion in https://github.com/openbraininstitute/neurodamus/pull/239#discussion_r2075323092, let's start to move all the imports to the top at the module level (except the NeuronWrapper).